core/src/main/java/org/kohsuke/stapler/interceptor/RequirePOST.java

```diff
@@ -31,11 +33,35 @@
 @Target({METHOD,FIELD})
 @InterceptorAnnotation(RequirePOST.Processor.class)
 public @interface RequirePOST {
+
+    /**
+     * Allows customizing the error page shown when an annotated method is called with the wrong HTTP method.
+     */
+    interface ErrorHandler {
+        ForwardToView getForwardView();
+    }
+
     public static class Processor extends Interceptor {
+        private static ErrorHandler handler;
```

Comment:
Maybe YAGNI, but it seems natural to have an option to register multiple handlers